solr/packaging/test/test_packages.bats

```diff
@@ -29,7 +29,12 @@ teardown() {
 }
 
 @test "package detects no running solr" {
-  # not sure this is actually a good thing..  we may not want this..
```

Comment:
I kind of want to keep this comment...   There is an issue open to look at this.   Why do we check if solr is running on this command?